src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -244,14 +244,10 @@ def _load_input_subgraphs(self,
 
             dangling_inputs = self._dangling_inputs(inputs, outputs)
             if dangling_inputs:
-                log.info('Subgraph %r has dangling inputs: %r', subgraph, dangling_inputs)
+                log.debug('Subgraph %r has dangling inputs: %r', subgraph, dangling_inputs)
```

Comment:
Both of these branches should result in some info-level log message. The message should be a summary for info and detailed for debug. I think you've implemented this pattern before. Search code base for `isEnabledFor(logging.DEBUG)` to find examples.

Reply:
Should there be a conditional, more detailed, debug-level message for both branches, or only the case where some dangling inputs are found?

Reply:
When there are no dangling inputs, what detail would you provide?

A note on process: a mere comment asking for clarification is only appropriate for PRs with normal urgency. I get 100 emails from Github everyday and am likely not to become aware of comments until I review a PR or empty out my inbox, which I only once a day (in this case it was lucky that I was just doing that). 

I can only guarantee that I see a question like this when I am reviewing the PR. So for urgent PRs you should reach out on Slack or request a review.